src/hamt/bitfield.js

```diff
@@ -0,0 +1,140 @@
+export const BITWIDTH = 256
+
+export const create = (bitCount = BITWIDTH) => new Uint8Array(bitCount / 8)
+
+/**
```

Comment:
could be helpful to have description of function or even just `@returns` w/ a description of what the return value means

Reply:
Ah this file is obsolete as it got replaced by `src/bitfield/*`, I'll just remove this